main/src/main/java/cgeo/geocaching/unifiedmap/tileproviders/TileProviderFactory.java

```diff
@@ -57,7 +57,10 @@ public static void addMapviewMenuItems(final Activity activity, final PopupMenu
             }
             if (!hide) {
                 final int id = tileProvider.getNumericalId();
-                parentMenu.add(tileProvider instanceof AbstractMapsforgeOfflineTileProvider ? R.id.menu_group_map_sources_offline : R.id.menu_group_map_sources_online, id, i, tileProvider.getTileProviderName()).setCheckable(true).setChecked(id == currentTileProvider);
+                final boolean isOfflineMap = (tileProvider instanceof AbstractMapsforgeOfflineTileProvider)
+                        || (tileProvider instanceof AbstractMapsforgeVTMOfflineTileProvider)
+                        || (tileProvider instanceof MapsforgeVTMMultiOfflineTileProvider);
```

Comment:
the last part (`tileprovider instanceof MapsforgeVTMMultiOfflineTileProvider`) is not required, as a `MapsforgeVTMMultiOfflineTileProvider` is a subclass of `MapsforgeVTMOfflineTileProvider`

Reply:
Removed